consent-ws/src/main/java/org/broadinstitute/consent/http/models/ConsentDataSet.java

```diff
@@ -9,12 +9,15 @@
     @JsonProperty
     public String consentId;
 
+    @JsonProperty
+    public String objectId;
+
     @JsonProperty
     public Map<String, String> dataSets;
-;
 
-    public ConsentDataSet(String consentId,  Map<String, String> dataSets) {
+    public ConsentDataSet(String consentId,  Map<String, String> dataSets, String objectId) {
         this.consentId = consentId;
+        this.objectId = objectId;
         this.dataSets = dataSets;
```

Comment:
what are the keys and values of this map? maybe a comment in the code.

Reply:
I hope the comment we've just added clarifies this.